cmd/argo/commands/watch.go

```diff
@@ -51,18 +51,17 @@ func watchWorkflow(wfName string) {
 	errors.CheckError(err)
 	for {
 		event, err := stream.Recv()
-		errors.CheckError(err)
-		wf := event.Object
-		if wf == nil {
+		errors.CheckError(err)		
```

Comment:
I believe `CheckError` will panic if `err` is not nil, so this code change is ineffective. Perhaps you meant to change
```suggestion
		if err != nil { ... }
```
?